addons/escoria-core/game/core-scripts/esc/commands/change_scene.gd

```diff
@@ -99,8 +106,9 @@ func run(command_params: Array) -> int:
 				if rc[0] != ESCExecution.RC_OK:
 					return rc[0]
 			
-			escoria.main.scene_transition.fade_in()
-			yield(escoria.main.scene_transition, "transition_done")
+			if !command_params[1]:
```

Comment:
This was here before, but I just saw, that if a room doesn't have any esc script attached, it will never be faded in again, because this code is inside the `if "esc_script"` clause.

Reply:
I added an else to manage the case where the room has no esc script attached. Thanks!